app/views/pages/catalogo/section_2.0.html

```diff
@@ -302,6 +302,12 @@ <h1>Proemio</h1>
 
             
          </section>
+         <a rel="prev"
+            class="btn btn-default btn-sm"
+            href="/catalogo/section_1.26">« Previous</a>
```

Comment:
Notice how it handles going to the previous section (1.26) from this section (2.0)... this was tricky to get working but the logic seems to be workinng.